contracts/core/liquidators/twap-impl/TWAPAuction.sol

```diff
@@ -142,32 +142,63 @@ contract TWAPAuction is TwoAssetPriceBoundedLinearAuction {
     )
         internal
     {
+        Auction.initializeAuction(
+            _twapAuction.chunkAuction.auction,
+            _currentSet,
+            _nextSet,
+            0
+        );
+
+        uint256 minimumBid = calculateMinimumBid(
+            _twapAuction.chunkAuction.auction,
+            _currentSet,
+            _nextSet
+        );
+
+        // Total currentSetQUantity
+        uint256 totalOrderSize = _startingCurrentSetQuantity.div(minimumBid).mul(minimumBid);
+        // uint256 totalOrderSize = _startingCurrentSetQuantity;
+
         // Calculate currency value of rebalance volume
         uint256 rebalanceVolume = LiquidatorUtils.calculateRebalanceVolume(
             _currentSet,
             _nextSet,
             oracleWhiteList,
-            _startingCurrentSetQuantity
+            totalOrderSize
         );
 
         // Calculate the size of each chunk auction in currentSet terms
         uint256 chunkSize = calculateChunkSize(
```

Comment:
Don't think this guarantees that each chunkSize is a multiple of a minimumBid. It guarantees that the totalOrderSize is a multiple of a minimumBid but lets say theres 3 chunks. Then each chunk is gonna have some remainder that's 1/3 (ish) of a minimumBid.

Reply:
hmm, that's a good point. Think we need to calculate a normalized chunkSize first then back into the total order size